probe/endpoint/reporter.go

```diff
@@ -14,6 +14,7 @@ import (
 
 // Node metadata keys.
 const (
+	Name = "name"
```

Comment:
Yes, I understand this change is just elevating a string literal to a const, but "name" is too generic to be reified in this way. What kind of name is it? Can we also change the string literal, or do we rely on that ambiguity in other places (I recall that we might)? If the answers to these questions are too difficult, as I suspect the might be, better to leave it as a string literal.